examples/terra-contract/Developing.md

```diff
@@ -19,6 +19,11 @@ rustup target list --installed
 rustup target add wasm32-unknown-unknown
 ```
 
+Also in order to make using and compiling the project easier please modify `pyth-sdk-terra` dependency in [Cargo file](./Cargo.toml)
```

Comment:
Suggested alternate wording:

> This example uses relative paths in Cargo.toml - You must remove any `path` components within `Cargo.toml` dependencies if you intend to compile this code outside of the `pyth-sdk-terra` repository, otherwise this will fail to compile. For example:

```diff
- pyth-sdk-terra = { version = "0.3.0", path = "../../pyth-sdk-terra" }
+ pyth-sdk-terra = { version = "0.3.0" }
```